### DISCUSSION TITLE
Can't run DistributionTool under macOS Ventura 13.3.1

### DISCUSSION THREAD

**Discussion by pozil** [elgatosf/elgato#15] [Help] [answered]
When running DistributionTool to build a custom plugin under macOS Ventura 13.3.1, I get an Apple security error message that I cannot bypass:

![Screenshot 2023-05-10 at 17 14 15](https://github.com/elgatosf/elgato/assets/5071767/9e307fef-bf56-4407-a073-9c7b4b6e126e)

@elgatosystems can you update the DistributionTool app so that it can be run with recent macOS version?

**Comment by ChekTek** [comment, marked as answer]
Hi!

Thank you for pointing this out! We are aware of the issue, but the DistibutionTool is planned to be retired soon. For now, you should be able to go into the macOS Privacy & Security preferences and allow the tool to run.

![CleanShot 2023-05-11 at 11 09 10](https://github.com/elgatosf/elgato/assets/6678919/3efc90c6-a280-4436-a0ad-ae58922e9d57)

**Reply by pozil** [reply]
Awesome, thanks for the quick reply. I'm looking forward to building cool stuff for my StreamDeck.
